### PR TITLE
feat: implement support of kebab case in property names

### LINKED ISSUE #82: generate fails on properties with kebab case
if specification contains properties in kebab generate fails

### PR DESCRIPTION
**Pull request by scink**
fix #82

### CODE REVIEW

File: src/language/typescript/common/data/serialized-type.ts [hunk 1]
```diff
@@ -214,14 +214,16 @@ export const getSerializedOptionType = (serialized: SerializedType): SerializedT
 export const getSerializedOptionalType = (isRequired: boolean, serialized: SerializedType): SerializedType =>
 	isRequired ? serialized : getSerializedOptionType(serialized);
 
+export const getValidatedPropertyName = (name: string): string => (name.includes('-') ? `['${name}']` : name);
```
Comment (raveclassic) [inline src/language/typescript/common/data/serialized-type.ts]: Is this helper used anywhere else but in `getSerializedPropertyType`? If not then let's move it there.

Comment (scink) [reply src/language/typescript/common/data/serialized-type.ts]: i mean, into the function?

Comment (raveclassic) [reply src/language/typescript/common/data/serialized-type.ts]: just inline it in `getSerializedPropertyType`

Comment (scink) [reply src/language/typescript/common/data/serialized-type.ts]: ammended

**Pull request merged**